todo.js

```diff
@@ -0,0 +1,34 @@
+var ToDo = function(title, description){
+    this.title = title;
+    this.description = description;
+    this.isDone = false;
+};
+
+var ToDoList = [];
+
+Array.prototype.addToDo = function(title, description){
```

Comment:
Can you create a custom object rather than attaching methods to Object prototype. This is usually a bad practice.
http://perfectionkills.com/extending-native-builtins/